src/applications/vaos/new-appointment/components/VAFacilityPage/getEligibilityMessage.js

```diff
@@ -74,15 +74,6 @@ export default function getEligibilityMessage({
   } else if (requestReason === ELIGIBILITY_REASONS.error) {
     title = 'We’re sorry. We’ve run into a problem';
     content = 'Something went wrong on our end. Please try again later.';
-  } else if (requestReason === ELIGIBILITY_REASONS.notSupported) {
-    title = 'This facility doesn’t accept online scheduling for this care';
-    content = (
-      <>
-        You’ll need to call your VA health facility to schedule this
-        appointment. Not all VA facilities offer online scheduling for all types
-        of care.
-      </>
-    );
```

Comment:
Removing this logic because if both direct schedule and request had facilities disabled, then it would be sent to the `noValidVAFacilities` component and hence never reach this if statement.

Reply:
Hmm I think this case also handles requestReason === ELIGIBILITY_REASONS.notSupported && directReason === ELIGIBILITY_REASONS.error. Why do we no longer need to account for that case?

Reply:
Good catch @JunTaoLuo! I'll revise the scenario and update the alert text for this ticket.